src/apps/vpn/mozillavpn.cpp

```diff
@@ -2236,3 +2238,27 @@ void MozillaVPN::registerExternalOperations() {
   eoh->registerExternalOperation(
       OpQuit, []() { MozillaVPN::instance()->controller()->quit(); });
 }
+
+void MozillaVPN::registerPushMessageTypes() {
+  PushMessage::registerPushMessageType(
+      "DEVICE_DELETED", [](const QJsonObject& payload) -> bool {
+        const QString& publicKey = payload["publicKey"].toString();
+        logger.debug() << publicKey;
```

Comment:
I think this is leftover from a long time ago when I was developing it. Would you mind removing this line?